tests/integration/testdata/start_api/main.py

```diff
@@ -73,6 +73,8 @@ def invalid_v2_respose_returned(event, context):
 def invalid_hash_response(event, context):
     return {"foo": "bar"}
 
+def invalid_body_response(event, context):
+    return {"errorType": "Error", "errorMessage": ""}
```

Comment:
Why mock this vs making the exception happen by throwing an exception. From the issue: that is what the customer is doing? https://github.com/iRoachie/http-api-example/blob/master/hello-world/app.js

Nothing strictly wrong with this but could become out of date. Will let you decide if this is enough of a reason to update.

Reply:
I think it is fine to leave the testing like this, this is mocking the what the client is calling on python instead of javascript, because this is the formatting for error on lambda https://docs.aws.amazon.com/lambda/latest/dg/nodejs-exceptions.html#nodejs-exceptions-syntax. So unless aws lambda changes the error formatting, It shouldn't become out of date.